include/exadg/acoustic_conservation_equations/time_integration/time_int_abm.h

```diff
@@ -127,6 +126,11 @@ class TimeIntAdamsBashforthMoulton
   std::shared_ptr<PostProcessorInterface<Number>> postprocessor;
 
   dealii::ConditionalOStream pcout;
+
+  // Store initially compute time step size. We do this because the value
+  // will not change, but we need adaptive time-stepping for efficient
+  // sub-time stepping methods.
+  double initial_time_step_size;
```

Comment:
I did not understand this comment when lookint at this in an isolated way. Maybe it becomes clear once I have seen the rest of the code.

Reply:
This comment is still not clear to me.

Reply:
For adaptive time stepping, we need `recalculate_time_step_size()`.

Since the time step size can not change as long as there is no background convection, `recalculate_time_step_size()` will always return the same value. To avoid computing the CFL time step size every time step, we simply store it as a member and return it.